src/Diagnostics/Samples/CSharp/CSharpAnalyzers/CSharpAnalyzers/ReadMe.txt

```diff
@@ -0,0 +1,27 @@
+﻿DESCRIPTION
```

Comment:
@srivatsn @tmeschter @JohnHamby This change is now ready for review: I have ported the analyzers and tests to VB, added comments and also addressed Sri's feedback.

You may want to directly start reviewing from this README.txt file and next set of source files for C# analyzers (VB analyzers are just ported versions of the C# analyzers, with absolutely no semantic differences).